### ISSUE TITLE
Svalbard not considered part of Norway

### ISSUE THREAD

**Issue by scenting** [CartoDB/data-services#165]
Svalbard is not considered part of Norway, although it shares the SJ ISO code with Jan Mayen, which is correctly considered Norway.

**Comment by iriberri** [comment]
Thanks for reporting, I'll take a look at it :-)

In France for example, we thoroughly split the France country from other regions like Corse, Martinique, or the French Guiana, which are geocodificable with their own ISO codes or names. I'll check how this case is being treated generally.

**Comment by iriberri** [comment]
Hi @scenting, I have just fixed the issue. I noticed that the behaviour wasn't consistent: Svalbard was separated but Jan Mayen not. Now, Svalbad and Jan Mayen live separatedly from Norway under the ISO3 code SJM.

Our intention was initially to separate these two regions from Norway, but some error in the calculations to separate the geometries provoked that some of them remained with Norway.

The piece of code in which we explicitly divide these regions is available here: https://github.com/CartoDB/data-services/blob/master/geocoder/admin0/sql/subdivide_polygons.sql
